### PR TITLE
Fei tiff

### PR DESCRIPTION
**Pull request by jburel**
Fix NPE see https://trac.openmicroscopy.org.uk/ome/ticket/12083

To test:
 * Open the image submitted via QA 7956 e.g. ```./showinf -omexml  PATH_TO_QA/7956/grid06-Montage5-2-Montage\ image.tif ```
 * Check that the resolution is correctly picked i.e. cm

### CODE REVIEW

**Comment by jburel** [comment]
--depends-on #1704

File: components/formats-gpl/src/loci/formats/in/FEITiffReader.java [hunk 1]
```diff
@@ -128,6 +127,9 @@ protected void initStandardMetadata() throws FormatException, IOException {
     String tag = ifds.get(0).getIFDTextValue(helios ? HELIOS_TAG : SFEG_TAG);
     tag = tag.trim();
 
+    if (tag.length() == 0) {
```
Comment (mtbc) [inline components/formats-gpl/src/loci/formats/in/FEITiffReader.java]: Just FYI Java 6 brings us `String.isEmpty()` too.

**Comment by dominikl** [comment]
Works like expected, good to merge.

**Comment by jburel** [comment]
--no-rebase

**Pull request merged**